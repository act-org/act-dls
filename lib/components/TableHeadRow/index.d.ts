/**
 * @prettier
 */
import * as React from 'react';
import { TableRowProps } from '@material-ui/core';
declare const TableHeadRow: React.FC<TableRowProps>;
export default TableHeadRow;
//# sourceMappingURL=index.d.ts.map