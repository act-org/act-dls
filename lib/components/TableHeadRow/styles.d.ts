/**
 * @prettier
 */
declare const _default: (props?: any) => Record<"tableRowRoot", string>;
export default _default;
//# sourceMappingURL=styles.d.ts.map