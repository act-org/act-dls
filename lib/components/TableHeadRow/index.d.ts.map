{"version":3,"file":"index.d.ts","sourceRoot":"","sources":["../../../src/components/TableHeadRow/index.tsx"],"names":[],"mappings":"AAAA;;GAEG;AAEH,OAAO,KAAK,KAAK,MAAM,OAAO,CAAC;AAC/B,OAAO,EAAY,aAAa,EAAE,MAAM,mBAAmB,CAAC;AAI5D,QAAA,MAAM,YAAY,EAAE,KAAK,CAAC,EAAE,CAAC,aAAa,CAazC,CAAC;AAEF,eAAe,YAAY,CAAC"}