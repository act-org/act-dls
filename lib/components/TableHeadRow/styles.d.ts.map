{"version":3,"file":"styles.d.ts","sourceRoot":"","sources":["../../../src/components/TableHeadRow/styles.ts"],"names":[],"mappings":"AAAA;;GAEG;;AAKH,wBAIG"}