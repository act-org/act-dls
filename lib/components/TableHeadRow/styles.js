"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;

var _colors = require("@material-ui/core/colors");

var _styles = require("@material-ui/core/styles");

/**
 * @prettier
 */
var _default = (0, _styles.makeStyles)({
  tableRowRoot: {
    backgroundColor: _colors.grey[800]
  }
});

exports["default"] = _default;