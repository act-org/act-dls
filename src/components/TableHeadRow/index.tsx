/**
 * @prettier
 */

import * as React from 'react';
import { TableRow, TableRowProps } from '@material-ui/core';

import useStyles from './styles';

const TableHeadRow: React.FC<TableRowProps> = (
  props: TableRowProps,
): React.ReactElement<any> => {
  const classes = useStyles();

  return (
    <TableRow
      {...props}
      classes={{
        root: classes.tableRowRoot,
      }}
    />
  );
};

export default TableHeadRow;
