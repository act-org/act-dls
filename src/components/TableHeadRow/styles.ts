/**
 * @prettier
 */

import { grey } from '@material-ui/core/colors';
import { makeStyles } from '@material-ui/core/styles';

export default makeStyles({
  tableRowRoot: {
    backgroundColor: grey[800],
  },
});
