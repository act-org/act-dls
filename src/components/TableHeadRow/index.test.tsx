/**
 * @prettier
 */

import * as React from 'react';

import render from '~/helpers/test/render';

import TableHeadRow from '.';

describe('TableHeadRow', () => {
  it('matches the snapshot', () => {
    const { container } = render(<TableHeadRow>child</TableHeadRow>);

    expect(container).toMatchSnapshot();
  });
});
