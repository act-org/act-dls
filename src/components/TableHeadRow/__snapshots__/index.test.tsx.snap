// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableHeadRow matches the snapshot 1`] = `
<div>
  <tr
    class="MuiTableRow-root makeStyles-tableRowRoot-1"
  >
    child
  </tr>
</div>
`;
